roles/openshift_health_checker/openshift_checks/package_version.py

```diff
@@ -9,6 +9,19 @@ class PackageVersion(NotContainerizedMixin, OpenShiftCheck):
     name = "package_version"
     tags = ["preflight"]
 
+    openshift_to_ovs_version = {
+        "3.6": "2.6",
+        "3.5": "2.6",
+        "3.4": "2.4",
```

Comment:
Thanks, handling this [here](https://github.com/openshift/openshift-ansible/pull/4064/files#diff-7a702243830dedaa3bae183335f1745aR122) now